src/test/java/guitests/KeyboardShortcutsTest.java

```diff
@@ -144,15 +144,48 @@ public void keyboardShortcutsTest() {
         clearPanelIndex();
 
         // remove focus from repo selector
-        ComboBox<String> comboBox = find("#repositorySelector");
-        doubleClick(comboBox);
-        assertEquals(true, comboBox.isFocused());
+        ComboBox<String> repoSelectorComboBox = find("#repositorySelector");
+        doubleClick(repoSelectorComboBox);
+        assertEquals(true, repoSelectorComboBox.isFocused());
         press(KeyCode.ESCAPE).release(KeyCode.ESCAPE);
-        assertEquals(false, comboBox.isFocused());
+        assertEquals(false, repoSelectorComboBox.isFocused());
         clearUiComponentFocusEventType();
         
         click("#dummy/dummy_col1_1");
 
+        // switch default repo tests
+        assertEquals(1, repoSelectorComboBox.getItems().size());
+        // setup - add a new repo
+        doubleClick(repoSelectorComboBox);
+        repoSelectorComboBox.getEditor().positionCaret(
+                repoSelectorComboBox.getValue().length()
+        );
+        for (int i = 0; i < repoSelectorComboBox.getValue().length(); i++) {
+            push(KeyCode.BACK_SPACE);
+        }
+        type("dummy1/dummy1");
+        push(KeyCode.ENTER);
+        PlatformEx.waitOnFxThread();
+        assertEquals(2, repoSelectorComboBox.getItems().size());
+        assertEquals(repoSelectorComboBox.getValue(), "dummy1/dummy1");
+        // test shortcut on repo dropdown
+        doubleClick(repoSelectorComboBox);
+        press(KeyCode.CONTROL).press(KeyCode.R).release(KeyCode.R).release(KeyCode.CONTROL);
+        PlatformEx.waitOnFxThread();
+        push(KeyCode.ESCAPE);
+        PlatformEx.waitOnFxThread();
+        assertEquals(repoSelectorComboBox.getValue(), "dummy/dummy");
```

Comment:
Something like [this](https://github.com/HubTurbo/HubTurbo/blob/master/src/test/java/guitests/UITest.java#L226)? I don't know whether the problem is that the repo dropdown is not in focus when the key press event happens though.

Anyway pushKeys is preferred to press/push. More convenient and there is less chance of problems with timing issues.